### DISCUSSION TITLE
关于GPT翻译功能使用问题反馈

### DISCUSSION THREAD

**Discussion by Nishino525** [Ayanaminn/N46Whisper#9] [General]
作者，你好。
本消息用于反馈问题。
问题现象：翻译完成后，重新从头开始翻译。
可能原因：（没有好好看源码，仅个人猜测）因为一些原因，中途电脑失去了一段时间的网络连接，再重新链接以后，可能存在部分语句翻译丢失问题，从而导致在翻译完成后，生成字幕文件时，校验出现问题从而导致重新从头翻译问题。
建议：是否可以考虑增加恢复工作区，起码不需要从头重新开始。增加对大文件的友好性，免去重复工作，造成时间的浪费。
对作者的付出表示感谢！

**Comment by Nishino525** [comment]
不好意思，后续发现谷歌云盘里是有翻译字幕文件输出的，因为当时没有下载至本地的提示，所以以为没有文件输出。不过重新翻译的问题是确实存在的。

**Comment by Ayanaminn** [comment]
这个问题有意思，可以请你再描述的详细一点吗？
翻译完成以后已经出现了完成的提示？然后又继续显示第一句的翻译？

**Reply by Nishino525** [reply]
因为耗时太长，所以并没有一直关注运行情况，没有抓到这一时刻。如果完成提示是弹窗的话，那并没有看到。
具体的情况是，前一次看离翻译完成不远了，过了一段时间再回来看发现翻译还在继续，仔细一看是又从头开始的。文本框是清除了数据的，重新开始的第一句就是在显示文本框的第一行。

**Reply by Ayanaminn** [reply]
完成以后在运行栏会有一个文本提示“双语字幕已经生成”
目前还没有复现这个问题，我觉得有可能是chatGPT那边掉线了...？
不过我会持续关注的，谢谢~
